### PR TITLE
Improve audit table display

### PR DESCRIPTION
**Pull request by omerzi**
- [x] All [tests](https://github.com/jfrog/jfrog-cli-core#tests) passed. If this feature is not already covered by the tests, I added new tests.
- [x] All [static analysis checks](https://github.com/jfrog/jfrog-cli-core/actions/workflows/analysis.yml) passed.
- [x] This pull request is on the dev branch.
- [x] I used gofmt for formatting the code before submitting the pull request.
-----
* Components changed to 'Direct Dependencies' 
* Impacted Package changed to 'Impacted Dependency'
* The rows now represent an Impacted Dependency separately for each CVE 

image

### CODE REVIEW

File: xray/utils/resultstable.go [hunk 1]
```diff
@@ -478,3 +483,175 @@ func getOperationalRiskViolationReadableData(violation services.Violation) *oper
...
+	}
+	// convert map to slice
+	result := make([]services.Vulnerability, 0, len(uniqueVulnerabilities))
+	for _, v := range uniqueVulnerabilities {
+		result = append(result, *v)
+	}
+	return result
+}
+
+func SimplifyViolations(scanViolations []services.Violation, multipleRoots bool) []services.Violation {
```
Comment (eyalbe4) [inline xray/utils/resultstable.go:521]: Let's add a description for this function.

File: xray/utils/resultwriter.go [hunk 2]
```diff
@@ -263,7 +298,7 @@ func findMaxCVEScore(cves []formats.CveRow) (string, error) {
 }
 
 // Splits scan responses into aggregated lists of violations, vulnerabilities and licenses.
-func splitScanResults(results []services.ScanResponse) ([]services.Violation, []services.Vulnerability, []services.License) {
+func SplitScanResults(results []services.ScanResponse) ([]services.Violation, []services.Vulnerability, []services.License) {
```
Comment (eyalbe4) [inline xray/utils/resultwriter.go:301]: Looks like this function can remain private.

File: xray/utils/resultwriter.go [hunk 3]
```diff
@@ -55,19 +61,19 @@ func PrintScanResults(results []services.ScanResponse, errors []formats.SimpleJs
...
+		jsonTable, err := convertScanToSimpleJson(results, errors, includeVulnerabilities, isMultipleRoots, includeLicenses, false)
 		if err != nil {
 			return err
 		}
 		return printJson(jsonTable)
 	case Json:
 		return printJson(results)
 	case Sarif:
-		sarifFile, err := GenerateSarifFileFromScan(results, includeVulnerabilities, isMultipleRoots)
+		sarifFile, err := GenerateSarifFileFromScan(results, includeVulnerabilities, isMultipleRoots, false)
```
Comment (eyalbe4) [inline xray/utils/resultwriter.go:76]: It looks like the `GenerateSarifFileFromScan` can be made private.

File: xray/utils/resultwriter.go [hunk 4]
```diff
@@ -31,14 +31,20 @@ const maxPossibleCve = 10.0
 
 var OutputFormats = []string{string(Table), string(Json), string(SimpleJson), string(Sarif)}
 
+type SarifProperties struct {
```
Comment (eyalbe4) [inline xray/utils/resultwriter.go:34]: It looks like this struct can be made private.

**Review by eyalbe4 — CHANGES_REQUESTED** [review]

File: xray/utils/resultstable.go (hunk 1 above)
Comment (omerzi) [reply xray/utils/resultstable.go:521]: done

File: xray/utils/resultwriter.go (hunk 2 above)
Comment (omerzi) [reply xray/utils/resultwriter.go:301]: done

File: xray/utils/resultwriter.go (hunk 4 above)
Comment (omerzi) [reply xray/utils/resultwriter.go:34]: done

File: xray/utils/resultwriter.go (hunk 3 above)
Comment (omerzi) [reply xray/utils/resultwriter.go:76]: Frogbot uses this function

**Pull request merged**